### PR TITLE
Update braker2 to 2.1.4

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`braker2`](https://bioconda.github.io/recipes/braker2/README.html): **2.1.2** &rarr; **2.1.4**

[install with bioconda](http://bioconda.github.io/recipes/braker2/README.html) [Conda](https://anaconda.org/bioconda/braker2/files)

Info | Link
-----|-----
Recipe | [`recipes/braker2`](https://github.com//bioconda/bioconda-recipes/tree/bump/braker2/recipes/braker2) (click to view/edit other files)
Releases |[https://github.com/Gaius-Augustus/BRAKER/tags](https://github.com/Gaius-Augustus/BRAKER/tags)
Author | `@Gaius-Augustus`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/braker2/path.patch [hunk 1]
```diff
@@ -1,62 +1,111 @@
...
  # When two gene sets are merged with joingenes, genes that
---- scripts/startAlign.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/startAlign.pl.new	2019-04-13 00:58:15.723573394 -0500
+diff --git a/scripts/gatech_pmp2hints.pl b/scripts/gatech_pmp2hints.pl
+index 22fc880..a00edd5 100755
+--- a/scripts/gatech_pmp2hints.pl
++++ b/scripts/gatech_pmp2hints.pl
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```
Comment (bgruening) [inline recipes/braker2/path.patch]: Why is this changed?

Comment (dpryan79) [reply recipes/braker2/path.patch]: So it uses the perl in the same environment.

Comment (bgruening) [reply recipes/braker2/path.patch]: A normal shebang also like the above will also use the same perl in the same environment if activated properly. Can you point me to this conda-forge recommendation? I don't think we should change code to make it anaconda compatible :( - However, we do encourage people since years to use a proper shebang. E.g. `/usr/bin/env perl` vs. `/usr/bin/perl`.

File: recipes/braker2/path.patch [hunk 2]
```diff
@@ -1,62 +1,111 @@
---- scripts/align2hints.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/align2hints.pl.new	2019-04-13 00:45:20.199730500 -0500
+diff --git a/scripts/align2hints.pl b/scripts/align2hints.pl
+index aeacb40..b64ae25 100755
+--- a/scripts/align2hints.pl
++++ b/scripts/align2hints.pl
 @@ -1,4 +1,4 @@
--#!/usr/bin/perl
+-#!/usr/bin/env perl
```
Comment (bgruening) [inline recipes/braker2/path.patch]: Here you haven't used the miniconda one?

File: recipes/braker2/path.patch (hunk 1 above)
Comment (dpryan79) [reply recipes/braker2/path.patch]: If environments are stacked then you otherwise get the perl from the last one activated, which may not be correct. When I come across people recommending this modified shebang again i'll let you know, I don't know that it was ever codified.

File: recipes/braker2/path.patch (hunk 2 above)
Comment (dpryan79) [reply recipes/braker2/path.patch]: I have, look at the next line down.
